docs/README.md

```diff
@@ -32,14 +32,25 @@ To use this app or override the default CSS you need import it in your dependenc
 
 Then, add `product-availability` block to your `blocks.json`
 
-Now, you can change the behavior of the `product-availability` block that is in the minicart. See an example of how to configure:
+Now, you can change the behavior of the `product-availability` block that is in the minicart. See an example of how to configure **without** showing availability:
```

Comment:
Is this file a duplicate of the root readme?

Reply:
yes, it is.

Reply:
The curren master branch has also this two versions of the readme file.

Reply:
Do you know why?

Reply:
I guess (but i'm not 100% sure) one belongs to the repo itself, and the other one is used for the Help Docs.